roles/dtc/common/templates/ndfc_attach_vrfs.j2

```diff
@@ -52,7 +52,6 @@
 {% for attach in MD_Extended.vxlan.overlay_services.vrf_attach_groups_dict[vrf['vrf_attach_group']] %}
     - ip_address: {{ attach['mgmt_ip_address'] }}
 {% endfor %}
-  deploy: false
 {% endif %}
-
+  deploy: false
```

Comment:
Spoke with Peter about this and this is probably ok.  Moves the `deploy` key outside of the attach check.

This:

```
{# ------------------------------------------------------ #}
{# Attach Group Section #}
{# ------------------------------------------------------ #}
{% if vrf['vrf_attach_group'] is defined %}
  attach:
{% for attach in MD_Extended.vxlan.overlay_services.vrf_attach_groups_dict[vrf['vrf_attach_group']] %}
    - ip_address: {{ attach['mgmt_ip_address'] }}
{% endfor %}
{% endif %}
  deploy: false
{% endfor %}
```
vs. this
```
{# ------------------------------------------------------ #}
{# Attach Group Section #}
{# ------------------------------------------------------ #}
{% if vrf['vrf_attach_group'] is defined %}
  attach:
{% for attach in MD_Extended.vxlan.overlay_services.vrf_attach_groups_dict[vrf['vrf_attach_group']] %}
    - ip_address: {{ attach['mgmt_ip_address'] }}
{% endfor %}
  deploy: false
{% endif %}
```